changelogs/fragments/8211-riak-admin-sub-command-support.yml

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+  - "riak - support ``riak admin`` sub-command in newer Riak KV versions beside the legacy ``riak-admin`` main command (https://github.com/ansible-collections/community.general/pull/8211)."
+  
```

Comment:
```suggestion

```

Reply:
Looks like there's a hidden space character on the last line that triggers the GitHub warning for no newline at the end of file.